app/frontend/src/application/jobseekers/location.js

```diff
@@ -10,21 +22,20 @@ export const onChange = (value) => {
   }
 };
 
-export const getCoords = () => {
-  if (INPUT_ELEMENT.dataset.coordinates) {
-    return INPUT_ELEMENT.dataset.coordinates.split(' ').join(',');
+export const getCoords = (el) => {
```

Comment:
Given we don't do anything with them, should we change the name of this method and make it return true or false?